evm/src/cpu/cpu_stark.rs

```diff
@@ -90,6 +128,60 @@ pub fn ctl_filter_logic<F: Field>() -> Column<F> {
     Column::sum([COL_MAP.op.and, COL_MAP.op.or, COL_MAP.op.xor])
 }
 
+pub fn ctl_sys_exc_check_rows<F: Field>() -> TableWithColumns<F> {
+    const OPS: [usize; 14] = [
+        COL_MAP.op.add,
+        COL_MAP.op.sub,
+        COL_MAP.op.mul,
+        COL_MAP.op.lt,
+        COL_MAP.op.gt,
+        COL_MAP.op.addfp254,
+        COL_MAP.op.mulfp254,
+        COL_MAP.op.subfp254,
+        COL_MAP.op.addmod,
+        COL_MAP.op.mulmod,
+        COL_MAP.op.submod,
+        COL_MAP.op.div,
+        COL_MAP.op.mod_,
+        COL_MAP.op.byte,
+    ];
+    // Create the CPU Table whose columns are the gas value and the rest of the columns are 0.
+    let filter = Some(Column::sum([COL_MAP.op.syscall, COL_MAP.op.exception]));
+
+    TableWithColumns::new(
+        Table::Cpu,
+        ctl_data_sys_exc_range_check(&OPS, &EXTRA_OPS),
+        filter,
+    )
+}
+
+pub fn ctl_prover_input_check_rows<F: Field>() -> TableWithColumns<F> {
+    const OPS: [usize; 14] = [
+        COL_MAP.op.add,
+        COL_MAP.op.sub,
+        COL_MAP.op.mul,
+        COL_MAP.op.lt,
+        COL_MAP.op.gt,
+        COL_MAP.op.addfp254,
+        COL_MAP.op.mulfp254,
+        COL_MAP.op.subfp254,
+        COL_MAP.op.addmod,
+        COL_MAP.op.mulmod,
+        COL_MAP.op.submod,
+        COL_MAP.op.div,
+        COL_MAP.op.mod_,
+        COL_MAP.op.byte,
+    ];
+    // Create the CPU Table whose columns are the gas value and the rest of the columns are 0.
+    let filter = Some(Column::single(COL_MAP.op.prover_input));
+
+    TableWithColumns::new(
+        Table::Cpu,
+        ctl_data_prover_input_range_check(&OPS, &EXTRA_OPS),
+        filter,
+    )
+}
+
```

Comment:
These can be deleted: we can reuse the base arithmetic CTL.

Reply:
@nbgl  The reason why I didn't reuse the base arithmetic CTL here was because I only selected the useful columns in the CPU for `prover_input` and `syscalls/exceptions`. For `prover_input`, the logic would still work with the base arithmetic CTL, but since we only wanted to range check `output[6]` for `syscalls` and `exceptions`, I thought we had to create an additional ctl. Am I missing something?
What I also thought we could do to avoid adding a CPU column is to range check `handler_addr0-2` and `syscall_info/exc_info`, since they're already in the columns selected by the base CTL, instead of only range checking the gas. What do you think?